docs/core/config/sieve/index.md

```diff
@@ -188,22 +189,26 @@ global scripts using the `sievec` command line tool. For example:
 sievec /var/lib/dovecot/sieve/global/
 ```
 
-This is necessary for scripts listed in [[setting,sieve_global]],
-[[setting,sieve_before]], and [[setting,sieve_after]] settings.
+This is necessary for script in storages with `after`, `before`, `global`, and
+`default` storage type (see [[link,sieve_storage_type]]).
```

Comment:
These would ideally be direct links also. You could create new sieve_storatge_type_after etc links.

Reply:
changed.